src/components/controls/AlertNotice.svelte

```diff
@@ -0,0 +1,73 @@
+<script>
+  import { fade } from 'svelte/transition';
+  import { cubicOut } from 'svelte/easing';
+
+  // So we don't pollute the entire LS namespace.
+  const KEY_PREFIX = 'alertNotice-';
+  export let dismissText = 'dismiss'; // Label of 'dismiss' button.
+
+  // Required: Which localStorage key to check whether this notice has been dismissed.
+  export let toggleKey = '';
+
+  $: alertVisible = window.localStorage.getItem(KEY_PREFIX + toggleKey) !== 'true';
```

Comment:
In Slack, Rob Miller mentioned that he'd like the message to reappear daily if it's not too much engineering effort, or if that would be too much engineering effort, to use `sessionStorage` instead.

Reply:
What does the `$:` reactivity do in this case? It doesn't seem that manually altering `localStorage` has any immediate effect.

Reply:
Good catch, just an artifact of some earlier testing.

Reply:
Just a remnant of an earlier approach, and yep I added the check against `today`